### ISSUE TITLE
Allow AoG.sorter( )

### ISSUE THREAD

**Issue by ReneSkukies** [unfoldtoolbox/UnfoldMakie.jl#232]
In AoG I can use the sorter function in case variables won't get sorted in the way I want (e.g. into subplots or inside a plot). Here is an example:

```julia
using Unfold, AlgebraOfGraphics

sorting = ['S', 'R']
	
eff = effects(Dict(:duration => 75:20:300), m)

data(eff) * visual(Lines) * mapping(:yhat, col = :eventname => sorter(sorting), color = :duration, group = :duration => nonnumeric) |> draw()
```

This gives me the following plot:
![image](https://github.com/user-attachments/assets/582ae0ae-b545-4555-a9ac-43708126b896)

However, in UnfoldMakie the following **does not** work:

```julia
using Unfold, UnfoldMakie

sorting = ['S', 'R']


eff = effects(Dict(:duration => 75:20:300), m)

plot_erp(eff;
	    mapping = (; col = :eventname => UnfoldMakie.AlgebraOfGraphics.sorter(sorting), color = :duration, group = :duration => nonnumeric)
	)
```

And without the sorting, the plot looks like this:
![image](https://github.com/user-attachments/assets/bdab7925-5b47-4070-ae99-437e3b0ae597)

Is it possible to include sorting in UM? Or is there already a way for this?

**Comment by ReneSkukies** [comment]
In case anyone runs into this problem: AoG (which is used for the plots by UnfoldMakie) seems to plot subplots in alphabetical order of the subplot name. Thus you _can_ get around the issue by renaming the stimuli in your model. 

For the example above, renaming stimuli to 'A' and 'R' would result in correctly plotted subplots.

**Comment by vladdez** [comment]
we maybe accidentially fixed this :shrug:

```julia
using Unfold, AlgebraOfGraphics

data,evts = UnfoldSim.predef_eeg()


evts
m = fit(UnfoldModel,["car"=>(@formula(0~1+continuous),firbasis((-0.1,1),100)),"face"=>(@formula(0~1+continuous),firbasis((-0.1,1),100))],evts,data;eventcolumn=:condition)	
eff = effects(Dict(:continuous => 75:20:300), m)
sorting = ["car","face"]

sorting = ["face","car"]

plot_erp(eff;
	    mapping = (; col = :eventname => sorter(sorting), 
        color = :continuous,group=:continuous)
	)
```
works now

![image](https://github.com/user-attachments/assets/44eb66dd-9b00-4a0d-aac4-eac9c6179346)

and 

![image](https://github.com/user-attachments/assets/5fa137db-a5c9-49ba-bd22-f36dad6d07ec)

**Comment by vladdez** [comment]
fixed #237
